docs/user-guide/helm.md

```diff
@@ -25,6 +25,13 @@ spec:
     namespace: kubeseal
 ```
 
+!!! note "When using multiple methods"
+    Order of precedence is `parameters > valuesObject > values > valueFiles > helm repository values.yaml` (see [this stackoverflow](https://stackoverflow.com/questions/76739856/what-is-the-precedence-setting-of-various-helm-values-in-an-argocd-application-m) for a more detailed example)
```

Comment:
Could we add the example here directly, instead of linking to stackoverflow?
cc @crenshaw-dev

Reply:
happy to do that,

Just felt it was a bit loud how about we reduce what the user sees for 
```
(see [this stackoverflow](https://stackoverflow.com/questions/76739856/what-is-the-precedence-setting-of-various-helm-values-in-an-argocd-application-m) for a more detailed example)
```
to
```
[Detailed Example](https://stackoverflow.com/questions/76739856/what-is-the-precedence-setting-of-various-helm-values-in-an-argocd-application-m)
```

Reply:
The issue with having a link is that the information can get stale, or the site might be down, or the link might not exist anymore. Having an example directly in the docs is also more easy to parse for a user than needing to dig through a thread (even if in this particular case it's just one answer).

Reply:
Agreed @phyzical, lets keep docs more self-contained.

Reply:
no worries will update now 👍